playbooks/roles/minishift/tasks/init_minishift.yml

```diff
@@ -1,19 +1,37 @@
 ---
 # start minishift
 - name: "Initialization of minishift cluster with profile {{ profile }}"
-  shell: "{{ minishift_bin }} start --profile {{ profile }} --disk-size {{ disk_size }} --memory {{ memory }} --openshift-version {{ oc_version }} --iso-url file:///{{ minishift_dest_dir }}/minishift.iso"
+  shell: "{{ minishift_bin }} start --profile {{ profile }} --cpus {{ cpus }} --disk-size {{ disk_size }} --memory {{ memory }} --openshift-version {{ oc_version }} --iso-url file:///{{ minishift_dest_dir }}/minishift.iso"
 
-- name: Wait for the cluster to come up completely
+- name: "Check if <cpu mode=host-passthrough/> is already set"
```

Comment:
Would it be better to put these tasks in a block that only has the 1 linux check vs having that check 4 separate times?

Reply:
Yes, that sounds reasonable. Will fix it.